### PR TITLE
Feature/update pipeline agent

### PR DESCRIPTION
**Pull request by satenderrathee**
updated agent in pipelines

### CODE REVIEW

**Review by RhysBushnell — DISMISSED** [review]

File: ingest/ingest-config-pipeline.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ trigger:
 
 
 pool:
-  vmImage: 'ubuntu-latest'
+  name: 'amido-stacks-core-data-agent-pool'
```
Comment (Trishisingh) [inline ingest/ingest-config-pipeline.yml:27]: should we add this as variable rather than value directly ?

Comment (satenderrathee) [reply ingest/ingest-config-pipeline.yml]: updated with variable

**Review by RhysBushnell — APPROVED** [review]

**Pull request merged**